### PR TITLE
feat(admin): document the new out-of-office feature

### LINKED ISSUE #11332: Out of office docs for admins
https://github.com/nextcloud/server/issues/40397

### PR DESCRIPTION
**Pull request by st3iny**
### ☑️ Resolves

* Fix #11332 

### 🖼️ Screenshots

I'm not sure if it really belongs to the groupware section. Suggestions for better locations are welcome.

grafik

### CODE REVIEW

File: admin_manual/groupware/out_of_office.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    occ config:app:set --value=yes dav hide_absence_settings
+
+.. note::
+
+    Out-of-office periods that were scheduled before the feature was disabled will not be deleted.
+    Disabling it will only hide the feature from the user interface. If the feature is enabled
+    again, the periods will be visible again.
+
+Set its value to `no` or delete it entirely to enable the feature again. The following
+command can be used to do so:
```
Comment (miaulalala) [inline admin_manual/groupware/out_of_office.rst:32]:
```suggestion
Set the value for `hide_absence_settings` to `no` or delete it entirely to enable the feature again. The following
command can be used to do so:
```

File: admin_manual/groupware/out_of_office.rst [hunk 2]
```diff
@@ -0,0 +1,36 @@
+=====================
+Out-of-office feature
+=====================
+
+.. versionadded:: 28.0.0
+
+The out-of-office feature allows users to schedule an out-of-office period including a status and
+message that is integrated with other apps such as Nextcloud Mail. Please refer to the user
+documentation for more information about the feature itself. It may be disabled globally by admins.
```
Comment (miaulalala) [inline admin_manual/groupware/out_of_office.rst:9]: Does a link exist for the user docs? You could insert it here.

**Review by miaulalala — CHANGES_REQUESTED** [review]
Some small nitpicks

File: admin_manual/groupware/out_of_office.rst (hunk 2 above)
Comment (st3iny) [reply admin_manual/groupware/out_of_office.rst:9]: Not yet but we can only link inside the same documentation, AFAIK. So I can't cross-link to the user doc from the admin doc.

Do you know if that is possible?

Comment (ChristophWurst) [reply admin_manual/groupware/out_of_office.rst:9]: It's not. We can only use absolute links to the deployment host.

**Review by miaulalala — APPROVED** [review]

**Pull request merged**